From a279c17434fbeaf040a553b6ce856ccbd00580f7 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Tue, 12 Dec 2017 14:06:13 +0100
Subject: [PATCH 64/82] ARM: dts: sun8i-a83t-tbs-a711: Enable
 force-poll-vbus-id-det on usbphy

A83T doesn't fire interrupts on PH11. Force polling.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts b/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts
index ea7b8bf4d327..028dd29e13d7 100644
--- a/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts
+++ b/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts
@@ -666,5 +666,6 @@
 	usb0_vbus-supply = <&reg_drivevbus>;
 	usb1_vbus_supply = <&reg_vmain>;
 	usb2_vbus_supply = <&reg_vmain>;
+	force-poll-vbus-id-det;
 	status = "okay";
 };
-- 
2.20.1

